tests/test_notebooks.py

```diff
@@ -40,6 +44,13 @@ def test_notebook(test_notebook):
     )
     ep.skip_cells_with_tag = "pytest-skip"
 
+    if INCLUDE_GPU_TESTS:
+        log.info(
+            "Note that only CPU tests will be run. In order to run tests that use an NVIDIA GPU,"
+            "use the `--include-gpu-tests` flag e.g. `tox -e kubeflow-local -- --include-gpu-tests`."
+            " To learn more, use `--help` or refer to the repository's README file."
+        )
```

Comment:
I think it should be quite the opposite, if gpu tests are not included we log this message